pages/common/upt.md

```diff
@@ -0,0 +1,38 @@
+# upt
+
+> Unified interface for managing packages across various operating systems, like Windows, many Linux distributions, Android, macOS, FreeBSD and even Haiku.
```

Comment:
```suggestion
> Unified interface for managing packages across various operating systems, like Windows, many Linux distributions, macOS, FreeBSD and even Haiku.
```

Removed Android. The project mentions termux packages in Android, but termux is a third party tool and it does not get permissions to touch core android binaries. pkg packages are only meant for termux.